graphql_compiler/compiler/compiler_frontend.py

```diff
@@ -83,7 +83,7 @@
     validate_root_vertex_directives, validate_vertex_directives,
     validate_vertex_field_directive_in_context, validate_vertex_field_directive_interactions
 )
-from .filters import process_filter_directive
+from .filters import _is_tag_argument, process_filter_directive
```

Comment:
Should `_is_tag_argument` be moved / made public somewhere? what place would be good for this?

Reply:
Perhaps either `metadata.py` or `helpers.py`, but move it only together with its dual, `is_variable_argument()`.